molli/storage/collection.py

```diff
@@ -53,7 +53,7 @@ def __init__(
     ) -> None:
         self._path = Path(path)
 
-        if not self._path.is_file() and readonly:
```

Comment:
This may need to be addressed in the future. I can definitely see how extension-less files and directories can conflict here, but let's hope for the best.
Non-issue for now.